core/src/main/java/io/aiven/kafka/tieredstorage/config/RemoteStorageManagerConfig.java

```diff
@@ -207,6 +214,21 @@ public class RemoteStorageManagerConfig extends AbstractConfig {
             ConfigDef.ValidList.in(SegmentCustomMetadataField.names()),
             ConfigDef.Importance.LOW,
             CUSTOM_METADATA_FIELDS_INCLUDE_DOC);
+
+        CONFIG.define(
+            UPLOAD_RATE_LIMIT_BYTES_CONFIG,
+            ConfigDef.Type.INT,
+            null,
+            Null.or(ConfigDef.Range.atLeast(1024 * 1024)), // at least 1MiB
```

Comment:
Not that we probably want practically go below this limit, but is there other reasons? It's anyway limited by the chunk size

Reply:
Based on this, https://github.com/Aiven-Open/tiered-storage-for-apache-kafka/pull/548#discussion_r1616049064 we won't be using chunk size for the time being -- but 1MiB looks like a good lower bound. Adding more comments and docs. Let me know if you agree.

The max lower bound could be 8KiB as the min rate -- but should be ok to push users to stay above 1MiB to guarantee some fair performance.